src/Authenticate.tsx

```diff
@@ -49,28 +51,106 @@ type Props = OwnProps & RouteComponentProps & ReduxProps
 const FETCH_WAIT = 60000
 
 @ErrorHandling
-export class Signin extends PureComponent<Props, State> {
+class AuthenticateUnconnected extends PureComponent<Props, State> {
   public state: State = {
     loading: RemoteDataState.NotStarted,
-    auth: false,
+    isAuthenticated: false,
   }
 
   private hasMounted = false
   private intervalID: NodeJS.Timer
 
+  private handleAuthenticationFailure = error => {
```

Comment:
this behavior used to be part of `checkForLogin`. It has been pulled out into a reusable method